rfcs/README.md

```diff
@@ -0,0 +1,31 @@
+# RFCs
+
+## Table of Contents
+
+<!-- toc -->
+- [What is an RFC and when should I create one?](#what-is-an-rfc-and-when-should-i-create-one)
+- [Creating a new RFC](#creating-a-new-rfc)
+- [Lifecycle of an RFC](#lifecycle-of-an-rfc)
+<!-- /toc -->
+
+This directory contains the "RFCs" for the `devbase` project. An RFC is a "Request for Comments". These are documents that describe the design and implementation of features in the `devbase` project.
+
+## What is an RFC and when should I create one?
+
+An RFC is a document that describes a design and implementation of a feature in the `devbase` project. It is intended to be a living document, and is updated as the scope of the feature/fix evolves over time.
+
+An RFC is created when a feature or fix is deemed to be large enough to warrant a more detailed discussion. This is to ensure that the design and implementation of the feature/fix is well thought out, documented, and given time for feedback from the community.
+
+## Creating a new RFC
+
+  1. Copy the `0-template.md` file to `rfcs/0000-my-feature.md` (where "my-feature" is a hyphenated version of the title. Don't assign an RFC number yet).
+  2. Fill in the RFC. Put care into the details: RFCs that do not; present convincing motivation, demonstrate understanding of the impact of the design, or are disingenuous about the drawbacks or alternatives tend to be poorly-received. You might want to create a [Draft Pull Request](https://github.blog/2019-02-14-introducing-draft-pull-requests/) to make this process easier and more effective.
+  3. Submit a pull request. As a pull request the RFC will receive design feedback from the larger community, and the author should be prepared to revise it in response.
+
+## Lifecycle of an RFC
```

Comment:
Would be worth including a small section about when an RFC is considered complete. Like if the functionality described in the RFC has been merged in, does that mean that the document is no longer living?

I think that we need a complete state and an ability to allow further amending (an additional section to describe major post release issue, lessons learned, etc.).

Reply:
I think this is done, please feel free to unresolve if not!